### DISCUSSION TITLE
Documents hard to read

### DISCUSSION THREAD

**Discussion by paul-vd** [react-hook-form/react-hook-form#1780] [General] [answered]
### After your latest update to the docs, it is really hard to read any syntaxes?

![image](https://user-images.githubusercontent.com/39598117/83801974-475ab700-a6aa-11ea-8d36-f32e771896a6.png)

> and it does not seem like a lot of thought was set into the theme?
## **light:**
![image](https://user-images.githubusercontent.com/39598117/83801997-53467900-a6aa-11ea-97de-72eda29d7477.png)

## **dark**
![image](https://user-images.githubusercontent.com/39598117/83802044-61949500-a6aa-11ea-8b8e-8a7c59e5cf2c.png)

## **light:**

![image](https://user-images.githubusercontent.com/39598117/83802066-6c4f2a00-a6aa-11ea-8590-462aa991c9bc.png)

## **dark**
![image](https://user-images.githubusercontent.com/39598117/83802095-75d89200-a6aa-11ea-9dcd-5963f3dc454c.png)

but yea the docs are really bad on the eyes... 👀

**Comment by bluebill1049** [comment, marked as answer]
We are demonstrating our support toward **Black Lives Matter**.

**Reply by paul-vd** [reply]
got it 💯 , any chance for a switch to display the syntax in color when reading a block?

**Reply by paul-vd** [reply]
nevermind I see it is back 😉

**Reply by bluebill1049** [reply]
yea, we had that theme on for 3 days to support **Black Lives Matter** and demonstrate our support towards stopping discrimination and racism. Just revert it back this afternoon.
